Essentials/src/com/earth2me/essentials/Teleport.java

```diff
@@ -247,6 +249,13 @@ protected void respawnNow(IUser teleportee, TeleportCause cause) throws Exceptio
     //The warp function is a wrapper used to teleportPlayer a player to a /warp
     @Override
     public void warp(IUser teleportee, String warp, Trade chargeFor, TeleportCause cause) throws Exception {
+        UserWarpEvent event = new UserWarpEvent(teleportee, warp, chargeFor);
```

Comment:
should be in its own PR?

Reply:
Yeah, I accidentally create the Papi branch out of the original branch of my fork and PR before even realize it.
We will just solve both PR at the same time or if anyone have a better idea.